src/test/java/AmazonTest.java

```diff
@@ -49,6 +50,30 @@ public void amazonTestOfPercentFunctionality() {
 
     }
 
+    @Test
+    public void checkPaginationAndHoveringOnBestSeller() {
+        CommonActions.createDriver(Config.PLATFORM_AND_BROWSER);
+        HomePage homePage = new HomePage(CommonActions.getDriver());
+        homePage.goTo();
+        homePage.waitForPageLoad();
+        homePage.chooseTopicButton();
+        homePage.waitForPreviousCategoryItemVisibility();
+        homePage.chooseCategory1("Smart Home");
+        homePage.waitForPreviousTopicItemVisibility();
+        homePage.chooseType("Smart Home Lighting");
+        SearchResultPage searchResultPage = new SearchResultPage(CommonActions.getDriver());
+        searchResultPage.waitForPageLoadVisibilityOfAllItem();
+        searchResultPage.seeAllResult();
```

Comment:
What does it mean "seeAllResult"? Do you understand what this method does?
Fix the name!!